locale/en-US/add-on.properties

```diff
@@ -28,13 +28,15 @@ sign_in_again_use_as_guest_button2=Use {{clientSuperShortname}} as a Guest
 panel_browse_with_friend_button=Browse this page with a friend
 panel_disconnect_button=Disconnect
 
-## LOCALIZATION_NOTE(first_time_experience_subheading2): Message inviting the
+## LOCALIZATION_NOTE(first_time_experience_subheading2, first_time_experience_subheading_button_above): Message inviting the
 ## user to create his or her first conversation.
 first_time_experience_subheading2=Click the Hello button to browse Web pages with a friend.
+first_time_experience_subheading_button_above=Click the button above to browse web pages with a friend.
```

Comment:
This string is unnecessary.  Take a look at the conversation in the bug between Matej & RT.